### ISSUE TITLE
<select> menu in mirror reverts to default option

### ISSUE THREAD

**Issue by LeaVerou** [bevacqua/dragula#456]
Testcase: http://codepen.io/leaverou/pen/QGPZoN

Drag the first item, note how the select menu reverts to item1 during drag. Same if you select a different value manually on any of the other items. 
`<input>`s don't have the same issue.

**Comment by bevacqua** [comment]
Could you work on the fix?

**Comment by LeaVerou** [comment]
If I had the time for that, this would've been a pull request, not an issue. :/

**Comment by NickolasBoyer** [comment]
It's the use of the cloneNode method ([here](https://github.com/bevacqua/dragula/blob/00ff5f9fa4ff62f09c5d16509dc290699344bec0/dist/dragula.js#L464)), this doesn't copy those properties of it's children. Perhaps using something like the innerHTML attribute would be an option?

_Current situation:_
`_mirror = _item.cloneNode(true);`

**Comment by LeaVerou** [comment]
Nope, `innerHTML` won't help, since runtime values don't affect the DOM. Something like this would (just a rough sketch):

```js
var selector = "select";  // more types can be added later
var cloneDescendants = _mirror.querySelectorAll(selector);
[...originalElement.querySelectorAll(selector)].forEach((element, i) => {
    cloneDescendants[i].value = element.value;
});
```

There might be a similar loop somewhere already, since `input` elements don't have the same bug.

**Comment by NickolasBoyer** [comment]
Ah stupid of course I can edit it and do a pull tomorrow if nobody is handling it currently?
